electron-app/package.json

```diff
@@ -3,24 +3,25 @@
   "name": "electron-app",
   "version": "0.0.0",
   "dependencies": {
-    "@theia/core": "^0.3.19",
-    "@theia/filesystem": "^0.3.19",
-    "@theia/workspace": "^0.3.19",
-    "@theia/preferences": "^0.3.19",
-    "@theia/navigator": "^0.3.19",
-    "@theia/process": "^0.3.19",
-    "@theia/terminal": "^0.3.19",
-    "@theia/editor": "^0.3.19",
-    "@theia/languages": "^0.3.19",
-    "@theia/markers": "^0.3.19",
-    "@theia/monaco": "^0.3.19",
-    "@theia/typescript": "^0.3.19",
-    "@theia/messages": "^0.3.19",
-    "@theia/textmate-grammars": "^0.3.19",
-    "pebble-extension": "0.0.0"
+    "@theia/core": "latest",
+    "@theia/filesystem": "latest",
+    "@theia/workspace": "latest",
+    "@theia/preferences": "latest",
+    "@theia/navigator": "latest",
+    "@theia/process": "latest",
+    "@theia/terminal": "latest",
+    "@theia/editor": "latest",
+    "@theia/languages": "latest",
+    "@theia/markers": "latest",
+    "@theia/monaco": "latest",
+    "@theia/typescript": "latest",
+    "@theia/messages": "latest",
+    "@theia/electron": "latest",
+    "@theia/textmate-grammars": "latest",
+    "fusion-studio-extension": "0.1.0"
```

Comment:
Should this not be `0.6.0` as that is what you set in `fusion-studio-extension/package.json`